test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9456,13 +9456,15 @@ def import_hook(self, name, body):
         """
         # sync_mom_hookfiles_timeout is 15min by default
         # Setting it to lower value to avoid the race condition at hook copy
-        srv_stat = self.status(SERVER,'sync_mom_hookfiles_timeout')
+        srv_stat = self.status(SERVER, 'sync_mom_hookfiles_timeout')
         try:
-           sync_val = srv_stat[0]['sync_mom_hookfiles_timeout']
+            sync_val = srv_stat[0]['sync_mom_hookfiles_timeout']
         except:
-           self.manager(MGR_CMD_SET,SERVER, {"sync_mom_hookfiles_timeout": 15})
+            self.manager(MGR_CMD_SET, SERVER,
+                         {"sync_mom_hookfiles_timeout": 15})
         else:
-           self.manager(MGR_CMD_SET,SERVER, {"sync_mom_hookfiles_timeout": sync_val})
+            self.manager(MGR_CMD_SET, SERVER,
```

Comment:
The way I'm reading this is that if server has 'sync_mom_hookfiles_timeout' value set, then set it again to the same value. Is this the intention?

Reply:
initially yes but I think I can just ignore it.